### ISSUE TITLE
Spike: Attempt fix a11y reads for collapsed nav menu 

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1984]
## Description

Nav items in the mobile/responsive version of TDP nav get ignored when NVDA (specific Windows-based screenreader) tries to read them. Possibly also affects Android's Talkback screenreader. We identified this issue during testing for #1937 but it also looked like it was introduced before then and just not caught.

## Action Taken

Tab through mobile nav items (from close to sign out and back) 

## What I expected to see

Each item should be read aloud by the screenreader

## What I did see

Screenshot of what NVDA was reading out. Each "tab" listed there represents a failed read of mobile nav menu items
![image](https://user-images.githubusercontent.com/28515874/190218577-928388f6-ef5e-4ea9-90ec-9fc1b7b0fe01.png)

## What to try
 
We've identified a likely culprit for this issue: 
```<div id="root" aria-hidden="true" data-nav-hidden="">```

Root having that aria-hidden cascades it to all elements on the page, including the sidenav. This property is basically a "please ignore this content" message to screen readers. Removing the aria-hidden from the root div or changing it to false should get us a working sidenav again! 

## Notes

My guess re: why the aria-hidden was added to begin with is to prevent screen readers in scan mode from reading outside of overlay elements on the page (e.g. the sidenav or timeout popups). Fixing this issue by removing or setting to false aria-hidden will result in screen readers being able to read page content while overlay elements are opened. This is also an issue that will need to be fixed, but it's a far less critical one than this issue.

If the above "What to try" is successful @reitermb will spin out a lower priority future enhancement to correct this less severe new issue (This will likely be a matter of adding more dynamically appearing aria-hidden="true" properties, just at lower levels of the page rather than the root level)

**Comment by stevenino** [comment]
@reitermb Can fully refine this ticket and tag Robert Jolly for review.

**Comment by reitermb** [comment]
Videos demonstrating blank/missing reads via NVDA & JAWS

https://user-images.githubusercontent.com/28515874/193977379-3d4dcf0e-611e-4c69-bd5e-803484821413.mp4
https://user-images.githubusercontent.com/28515874/193977386-2f61e7d6-016e-4d4e-bc1c-f9199544f12f.mp4

**Comment by raftmsohani** [comment]
![Screen Shot 2022-11-28 at 9.41.55 AM.png](https://images.zenhubusercontent.com/61d3697f7808855468eeaafa/8ac97662-cf0d-4d0d-824c-e820925894a4)

As it can be seen in the picture: removing the ```aria-hidden="true"``` corrected the issue.

**Comment by raftmsohani** [comment]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden

**Comment by lilybl1** [comment]
@raftmsohani Please document and create new ticket for introduced defect on this ticket. Thank you!

**Comment by stevenino** [comment]
Follow on tix #2307 has been created to address fix identified in this spike. Tix is ready to close.

**Comment by reitermb** [comment]
Adding evidence of a11y testing for this one too—also resolved by the USWDS version bump. 

https://github.com/user-attachments/assets/1bb92e62-24e8-4cda-aeda-e47bfe560640

cc @ttran-hub @ADPennington @lhuxraft @victoriaatraft

**Comment by ttran-hub** [comment]
Thanks Miles! a11y LGTM! 🚀
